pyomo/solvers/tests/checks/test_gurobi_persistent.py

```diff
@@ -309,3 +309,23 @@ def _my_callback(cb_m, cb_opt, cb_where):
         opt.solve()
         self.assertAlmostEqual(m.x.value, 1)
         self.assertAlmostEqual(m.y.value, 1)
+
+    @unittest.skipIf(not gurobipy_available, "gurobipy is not available")
+    def test_add_column(self):
+        m = pe.ConcreteModel()
+        m.x = pe.Var(within=pe.NonNegativeReals)
+        m.c = pe.Constraint(expr=(0, m.x, 1))
+        m.obj = pe.Objective(expr=-m.x)
+
+        opt = pe.SolverFactory('gurobi_persistent')
+        opt.set_instance(m)
+        opt.solve()
+        self.assertAlmostEqual(m.x.value, 1)
+
+        m.y = pe.Var(within=pe.NonNegativeReals)
+
+        opt.add_column(m, m.y, -2, [m.c], [1])
```

Comment:
I'd like to see this coefficient be something other than `1` (1 is easy to accidentally generate "passes" by having the coefficient actually be omitted).  Could you change this to 2? ... the same comment applies to the example in CPLEX and Xpress

Reply:
Done!